ui/app/appointments/directives/datePicker.js

```diff
@@ -22,18 +23,22 @@ angular.module('bahmni.appointments')
                 $scope.viewDate = $scope.viewDate && dateUtil.addDays($scope.viewDate, 1);
             };
 
-            $scope.$watch("viewDate", $scope.onChange($scope.viewDate));
-
+            $scope.$watch("viewDate", function (viewDate) {
+                $scope.viewDate = viewDate;
```

Comment:
Hi @smartyrad ,
Please remove this line

Reply:
@sumanmaity112 Changed that. Sorry for not having noticed it before